mozci/util/taskcluster.py

```diff
@@ -91,6 +142,26 @@ def get_task(task_id, use_proxy=False):
     return queue.task(task_id)
 
 
+def get_dependent_tasks_url(task_id):
```

Comment:
Would be more consistent to support passing the TC root url as in `get_indexed_tasks_url`

Reply:
Some calls have it in this file, some don't. But I will add the `root_url` anyway, it's more versatile.